engine/lite/components/kafka-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaUniversalAvroSchemaFunctionalTest.scala

```diff
@@ -87,7 +89,10 @@ class LiteKafkaUniversalAvroSchemaFunctionalTest
           sConfig(sampleBoolean, booleanSchema, integerSchema, sampleLong),
           invalidTypes(s"actual: '${typedLong.display}' expected: 'Integer'")
         ),
-        (sConfig(null, nullSchema, integerSchema, Input), invalidTypes("actual: 'Null' expected: 'Integer'")),
+        (
+          sConfig(null, nullSchema, integerSchema, Input),
+          emptyMandatoryParameter().findValid(invalidTypes("actual: 'Unknown' expected: 'Integer'"))
```

Comment:
Let's continue this https://github.com/TouK/nussknacker/pull/4887#discussion_r1371991324 discussion here

Reply:
Change in `MandatoryValidator` fixed that test, so this discussion is also resolved. :)